core/templates/dev/head/pages/exploration_player/TutorCardDirective.js

```diff
@@ -120,13 +128,18 @@ oppia.directive('tutorCard', [
           $scope.OPPIA_AVATAR_IMAGE_URL = (
             UrlInterpolationService.getStaticImageUrl(
               '/avatar/oppia_avatar_100px.svg'));
+
+
```

Comment:
Not sure why we're adding so many newlines here and below $scope.profilePicture?